.changeset/young-moons-deny.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/phrases-experience": minor
+"@logto/phrases": minor
+---
+
+add ar-AR language (credit to @zaaakher)
```

Comment:
1. According to PR description, this should be `ar-SA` instead?
2. Also I think `ar-SA` might not be the correct one, as I did some research on Google,  it shows the language code `ar-SA` a variant for Saudi Arabia, while `ar` is the generic one for Arabic. BTW, Chrome browser uses `ar` for Arabic.
3. There's no language code called `ar-AR`, according to: http://www.lingoes.net/en/translator/langcode.htm